docs/Classes.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
```

Comment:
Is it a rendered HTML by jazzy? If so, don’t add them to main branch. Just build it then pushing to gh-pages branch in workflow.